rust/main/utils/run-locally/src/cosmos/termination_invariants.rs

```diff
@@ -114,7 +114,7 @@ pub fn termination_invariants_met(
     if !provider_metrics_invariant_met(
         &relayer_metrics_port.to_string(),
         messages_expected,
-        &hashmap! {"status" => "success"},
+        &hashmap! {"connection" => "rpc", "status" => "success"},
```

Comment:
we should add a check for grpc in the invariants as well

Reply:
Good call 👍